pkgs/development/rocm-modules/6/hipfort/default.nix

```diff
@@ -19,6 +20,15 @@ stdenv.mkDerivation (finalAttrs: {
     hash = "sha256-Nks1+0X8bLtZ9HqZXJOtrAWQlJquMH+feuu1stf/9Vo=";
   };
 
+  patches = [
+    (fetchpatch {
+      name = "unify-cmake-minimum-version.patch";
```

Comment:
I guess it doesn’t really “unify” now, right? I’d probably just call this `[hipfort-]fix-cmake-4.patch` or something. Non‐blocking, though.